client/src/locale/en.js

```diff
@@ -1548,7 +1548,7 @@ const en = {
         expiryDate: "Invitation expires on",
         expiryDateTooltip: "The expiration date of the invitation, after which the invitation can no longer be accepted.",
         membershipExpiryDate: "Membership valid until",
-        membershipExpiryDateTooltip: "The expiration date of the membership, after which the user cannot log in to services connected to this collaboration.",
+        membershipExpiryDateTooltip: "The expiration date of the membership, after which the user cannot log in to applications connected to this collaboration.",
```

Comment:
maybe a good chance to standardize on "cannot" vs "can not". 

Mostly we are using "can not" which is a little awkward to me, but if it is the preferred way that's okay too.